packages/react-composites/review/react-composites.api.md

```diff
@@ -753,6 +766,62 @@ export type TopicChangedListener = (event: {
     topic: string;
 }) => void;
 
+// @beta (undocumented)
+export const UPLOAD_CANCELLED_EVENT = "uploadCancelled";
+
+// @beta (undocumented)
+export const UPLOAD_COMPLETED_EVENT = "uploadCompleted";
+
+// @beta (undocumented)
+export const UPLOAD_FAILED_EVENT = "uploadFailed";
+
+// @beta (undocumented)
+export const UPLOAD_PROGRESSED_EVENT = "uploadProgressed";
+
+// @beta
+export type UploadCanceledListener = () => void;
+
+// @beta
+export type UploadCompleteListener = (metaData: FileMetaData) => void;
+
+// @beta
+export class UploadedFile implements UploadEvents {
```

Comment:
The naming here is a bit confusing. It's called `UploadedFile`, but then has listners for file upload progress. Perhaps `FileUploadOperation`, or `ActiveFileUpload`, or even `FileUpload` (then methods could read like `FileUpload.cancel()`, `FileUpload.on('cancelled', <handler>` etc.